open_event/templates/gentelella/admin/footer.html

```diff
@@ -17,14 +16,14 @@ <h4 class="footer-header">Use Open Event</h4>
                 <ul>
 
                     {% for event_type in event_typo%}
-                    <li><a href="/how_it_works">{{event_type}}</a></li>
+                    <li><a href="{{ url_for('admin.browse_view') }}?type={{ event_type | urlencode }}">{{event_type}}</a></li>
```

Comment:
You can pass type to url_for, so wouldn't need to hard code it.

```python
url_for('admin.browse_view', type=event_type)
```

URL encoding is done automatically.